src/test/java/com/google/sps/api/CalendarClientHelperTest.java

```diff
@@ -31,19 +33,23 @@
 
   private static final String ACCEPTED = "accepted";
   private static final String DECLINED = "declined";
+  public static final String DATE_WITHOUT_TIME = "2019-06-20";
+  public static final String RFC3339_WITH_TIME = "2019-10-12T07:20:50.52Z";
   private EventAttendee ATTENDING_SELF = new EventAttendee();
   private EventAttendee DECLINED_SELF = new EventAttendee();
   private EventAttendee NOT_RESPONDED_SELF = new EventAttendee();
   private EventAttendee ATTENDING_GUEST = new EventAttendee();
   private EventAttendee DECLINED_GUEST = new EventAttendee();
   private EventAttendee NOT_RESPONDED_GUEST = new EventAttendee();
-
+  private List<EventAttendee> allSelfAttendeeds = Arrays.asList(ATTENDING_SELF, DECLINED_SELF, NOT_RESPONDED_SELF);
+  private List<EventAttendee> allGuestAttendeeds = Arrays.asList(ATTENDING_GUEST, DECLINED_GUEST, NOT_RESPONDED_GUEST);
+  public static final long EPOCH_TIMEPOINT = 1619827200;
 
   @Before
   public void setAttendees() {
-    ATTENDING_SELF.setSelf(true);
-    DECLINED_SELF.setSelf(true);
-    NOT_RESPONDED_SELF.setSelf(true);
+    allSelfAttendeeds.forEach((attendee)-> new EventAttendee());
```

Comment:
I'm sorry, but that would work as you expect. The newly created instance would be dropped and not assigned anywhere. There is no way around `ATTENDING_SELF = new EventAttendee();` etc.

Reply:
That's a shame

Reply:
Why does it happen?

Reply:
Ahh okay I guess I understand, the code is called in the lambda, but it doesn't change anything just gives a return value of a new EventAttendee, which is thrown away.

Reply:
Yep, exactly.